aspnetcore/tutorials/first-mvc-app/adding-controller.md

```diff
@@ -31,19 +31,19 @@ We cover these concepts in this tutorial series and show you how to use them to
 # [Visual Studio](#tab/visual-studio)
 
 * In the **Solution Explorer**, right-click **Controllers > Add > Controller**
-  ![Solution Explorer, right click Controllers > Add > Controller](~/tutorials/first-mvc-app/adding-controller/_static/add_controllerCopy.png)
+  ![Solution Explorer, right click Controllers > Add > Controller](~/tutorials/first-mvc-app/adding-controller/_static/5/add_controllerCopy.png)
 
 * In the **Add Scaffold** dialog box, select **MVC Controller - Empty**
 
-  ![Add MVC controller and name it](~/tutorials/first-mvc-app/adding-controller/_static/acCopy.png)
+  ![Add MVC controller and name it](~/tutorials/first-mvc-app/adding-controller/_static/5/acCopy.png)
```

Comment:
Same as previous suggestion. It's a bug in the system that it didn't render the correct image. If the original image doesn't work, delete it, but keep a name change so we don't hit this bug.

Reply:
Thanks Rick!  ( Just a heads up, this is in draft and not ready for review.  I'm still pushing lots of updates to many files.)

You asked why move the screenshots:  I was not moving the existing, those need to stay, I created new version 5 specific screenshots where needed related to the latest VS, while the earlier versions need to be maintained for the VS/SDK in prereqs for those versions.

Reply:
That's not 5 specific, that's VS specific . This block is in moniker 3 and greater. My file was created with the .NET 5 and the current VS.

I realize it's in draft.

Reply:
@Rick-Anderson.  Yes, I understand the screenshot is not 5 specific of course, it is specific to the VS prereq we have on the version of the docs which are versioned by SDK, not IDE.   I saw many examples of screenshots grouped to the doc sdk version and was following that pattern.  Sounds like I shouldn't.

How do you want to specify differences between VS and VC and VS for Mac versions for screenshots?  I can put it in the screenshot name, or I can create new directories for each, but that gets a bit busy.

It looked like there was minor change in VS 16.8.  They were minor, but you can see the difference, and it was a little confusing to see a different one so I updated it. acCopy: Controller on the left menu moved under MVC.  It was not at the top level, for me anyway.  How are we maintaining a VS 2017 for previous docs where that is the prereq?  Are we just paving over?  I saw many examples in the repo where we are maintaining them, so I assumed that was the pattern to follow and it seemed to make sense.

I will also go back and correct for moniker version, thanks.

Reply:
> I will also go back and correct for moniker version, thanks.

No, that image should work for both versions.

> How do you want to specify differences between VS and VC and VS for Mac versions for screenshots? I can put it in the screenshot name

That works.

Reply:
Here is acCopy so the differences are easy to see.   The dialog name and menu are different.
![image](https://user-images.githubusercontent.com/10985336/99006234-fb345e00-24f6-11eb-9c9e-63f1096a40b4.png)

Reply:
That one for sure needed an update, the previous, less so.